nixos/modules/programs/iay.nix

```diff
@@ -6,7 +6,7 @@ let
 in {
   options.programs.iay = {
     enable = mkEnableOption (lib.mdDoc "iay");
-    package = mkPackageOption pkgs "iay" {};
+    package = mkPackageOption pkgs (lib.mkDoc "iay") {};
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** isn't `mdDoc`?

Reply:
See [this](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/development/option-declarations.section.md#mkpackageoption-mkpackageoptionmd-sec-option-declarations-util-mkpackageoption).

Reply:
```suggestion
    package = mkPackageOptionMD pkgs "iay" {};
```